openssl_source/script/byte_asm_scan.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+
+# jan 30, 2020
+# markg
```

Comment:
Suggest refining the comments, e.g.:
#Jan 30, 2020
#Gentry, Mark <mark.gentry@intel.com>